app/Http/Controllers/ProfileController.php

```diff
@@ -216,4 +219,49 @@ public function getMenuState(Request $request)
             $request->session()->put('menu_state', 'closed');
         }
     }
+
+
+    /**
+     * Print inventory
+     *
+     * @author A. Gianotto
+     * @since [v6.0.12]
+     * @return \Illuminate\Http\RedirectResponse
+     */
+    public function printInventory()
+    {
+        $show_user = User::where('id', Auth::user()->id)->withTrashed()->first();
+        $assets = Asset::where('assigned_to',  Auth::user()->id)->where('assigned_type', User::class)->with('model', 'model.category')->get();
```

Comment:
It's a pretty minor point but I think if we're using the right polymorphic type here you can say something `Asset::where('assignedTo', Auth::user())` - or something like that - (losing the `assigned_type` bit). Not enough for me to reject a PR tho